src/kubectlUtils.ts

```diff
@@ -189,16 +189,18 @@ export async function getPods(kubectl: Kubectl, selector: any, namespace: string
         labelStr = "--selector=" + labels.join(",");
     }
 
-    const pods = await kubectl.asJson<KubernetesCollection<Pod>>(`get pods -o json ${nsFlag} ${labelStr}`);
+    const pods = await kubectl.fromLines(`get pods -o wide ${nsFlag} ${labelStr}`);

```

Comment:
What do we gain by changing this?  `asJson` should still be able to give us a status I think (via `spec.status`?) - is there a reason to prefer switching to `fromLines`?  (To be clear, I think the change is harmless, but I have an aversion to change for change's sake.)

Reply:
The specific reason to change this to `fromLines` was to get aggregated pod status which is very handy to add the status indicator icons. `asJson` returns `spec.status` which would say running even if one of the pods has in a crash loop state. It would be possible to get the same aggregated status from `asJson` but it involves aggregating the status manually by implementing something similar to `kubectl` [printer](https://github.com/kubernetes/kubernetes/blob/master/pkg/printers/internalversion/printers.go#L547), I did go down this route but found the solution to be convoluted hence I preferred `fromLines`